src/background/api/txHelper.js

```diff
@@ -299,7 +299,7 @@ export async function buildParatimeTxBody(params, wrapper) {
         oasis.quantity.fromBigInt(amount),
         oasisRT.token.NATIVE_DENOMINATION
     ]);
-    let feeAmount  = params.feeAmount||0
+    let feeAmount  = new BigNumber(params.feeAmount||0).multipliedBy(decimal).dividedBy(new BigNumber(10).pow(9)).toFixed()
```

Comment:
cointypes.decimals might be better than a literal 9 here. the real reason the UI uses nano ROSE/TEST is because it uses `amountDecimals` without a paratime-specific decimals setting